### PR TITLE
Componente detalle de Novedad

### PR DESCRIPTION
**Pull request by MartinCaparros**
I create a new route in App.js Create two component, one that recieves a tittle by prop and renders on a new component, also it render new data that comes from a callback to the endpoint with the ID from the new. I also create the service that makes the callback to the endpoint wich takes ID as parameter.

link to ticket: https://alkemy-labs.atlassian.net/browse/OT91-49

### CODE REVIEW

File: src/Components/News/Detail/NewsDetailLayout.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { getNewById } from '../../../Services/getNewById';
+
+const NewsDetailLayout = ({prop}) => {
+
+    const [newData,setNewData] = useState({})
+
+    const {id} = useParams()
+
+    useEffect(()=>{
+        const loadNewData = async () => {
```
Comment (saralavanchy) [inline src/Components/News/Detail/NewsDetailLayout.js]: the useffect should not have functions declarations. please move this outside the useEffect

File: src/Services/getNewById.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
```
Comment (saralavanchy) [inline src/Services/getNewById.js]: this should go on news service

File: src/Components/News/Detail/NewsDetailLayout.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <img alt="newImg" src={newData.img}></img>
+            <Box>
+                {newData.body}
+            </Box>
+        
+        </div> 
+
+    );
+}
+
```
Comment (saralavanchy) [inline src/Components/News/Detail/NewsDetailLayout.js:34]: please, some general feedback:
- remove unnecessary spaces
- all the times that you can put the code in only one line.

Same feedback on src/Components/News/Detail/NewsTittle.js file.

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/News/Detail/NewsDetailLayout.js (hunk 1 above)
Comment (MartinCaparros) [reply src/Components/News/Detail/NewsDetailLayout.js]: Done! :)

File: src/Services/getNewById.js (hunk 2 above)
Comment (MartinCaparros) [reply src/Services/getNewById.js]: Done!

File: src/Components/News/Detail/NewsDetailLayout.js (hunk 3 above)
Comment (MartinCaparros) [reply src/Components/News/Detail/NewsDetailLayout.js:34]: Sory about that, already fix it.

File: src/App.js [hunk 4]
```diff
@@ -30,6 +31,7 @@ function App() {
           <Route path="/create-project" component={ProjectsForm} />
           <Route path="/school-campaign" component={SchoolCampaign} />
           <Route path="/toys-campaign" component={ToysCampaign} />
+          <Route path="novedades/:id" component={NewsDetailLayout} />
```
Comment (saralavanchy) [inline src/App.js]: the / is missing on the route

File: src/Components/News/Detail/NewsDetailLayout.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    
+    useEffect(()=>{
+        loadNewData()
+    },[id])
+    
+    return ( 
+        <div>
+            <NewsTitle
+                title={prop.title}
```
Comment (saralavanchy) [inline src/Components/News/Detail/NewsDetailLayout.js:25]: properties on the props. Its not a blocking change

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 4 above)
Comment (MartinCaparros) [reply src/App.js]: Already fix it, sory!

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**